src/main/java/seedu/address/storage/LibraryStorage.java

```diff
@@ -68,7 +68,7 @@ public LibraryStorage() {
      * @param bookTitle the book title of a book
      * @return true if the book title is valid, false otherwise
      */
-    private static boolean isValidBook(String bookTitle) {
+    public static boolean isValidBook(String bookTitle) {
```

Comment:
We have changed isValidBook method from private accessifier to public.

This change is does not change the code behavior.

The reason why we change this is to increase our code quality by increase our test cases.